clinical-domain-agent/src/test/java/care/smith/fts/cda/services/FhirResolveServiceIT.java

```diff
@@ -104,6 +145,20 @@ void emptyPatientIdThrows() {
         .withMessageContaining("empty");
   }
 
+  @Test
+  void hdsReturnsMoreThanOneResult() throws IOException {
+    var fhirResolveGen = resolveSearchResponse(() -> "id1", randomUuid());
+
+    var bundle = fhirResolveGen.generateResources().limit(2).collect(toBundle());
+
+    wireMock.register(
+        getBuilder()
+            .willReturn(
+                ok().withHeader("Content-Type", APPLICATION_FHIR_JSON)
```

Comment:
We have fhirResponse in MockServerUtil, it does this. We could add one without status code, defaulting to OK.